src/guide/extras/music.md

```diff
@@ -0,0 +1,108 @@
+# Music
+
+aoi.js has an extension named ***"@akarui/aoi.music"*** for music.
+
+## installation
+
+> ```
+> npm i @akarui/aoi.music
+> ```
+
+***for Dev Version***
+
+> ```
+> npm i https://github.com/akaruidevelopment/music#main
```

Comment:
@dev for @akarui/aoi.music will be soon.

Reply:
ok done